emqx/hatch.toml

```
@@ -0,0 +1,7 @@
+[env.collectors.datadog-checks]
+
+[[envs.default.matrix]]
+python = ["3.9"]
```

```suggestion
python = ["3.11"]
```

The agent 7.51+ is now shipped with python 3.11 so I think it would be better to bump this version